docs/requirements.txt

```diff
@@ -1,6 +1,6 @@
 Sphinx==3.4.3
```

Comment:
There is a mismatch between `depthai-docs-website` requirements (it was updated few days ago), and this one which could cause further issues.
https://github.com/luxonis/depthai-docs-website/blob/master/requirements.txt

Reply:
Yes, they were set to this version during pybind-mkdoc integration somewhere around a month ago. Since we had issue with styling, I didn't want to change other dependencies now, mostle because we're working on docs fixes now